assets/js/script.js

```diff
@@ -1,30 +1,28 @@
-let bookArray = [];
-const html = [];
-const myStorage = window.localStorage;
-bookArray = Object.entries(myStorage);
+class Book {
+  constructor() {
+    this.bookArray = Object.entries(window.localStorage);
+    this.html = [];
+  }
 
-// eslint-disable-next-line no-unused-vars
-function addBooks() {
-  const bookName = document.getElementById('name-book').value;
-  const bookAuthor = document.getElementById('author-book').value;
-  localStorage.setItem(bookName, bookAuthor);
-}
-
-function removeBooks(i) {
-  const title = i.firstChild.innerText;
-  Object.fromEntries(bookArray);
-  localStorage.removeItem(title);
-  bookArray = Object.entries(myStorage);
-  i.remove();
-}
+  addBooks() {
+    this.bookName = document.getElementById('name-book').value;
+    this.bookAuthor = document.getElementById('author-book').value;
+    localStorage.setItem(this.bookName, this.bookAuthor);
```

Comment:
[Optional]
You can use unique id property for each book. It would make it easier to remove books with duplicate values.